starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,62 @@
+body {
+  font-family: 'Open Sans', sans-serif;
+
+}
+
+/* Hero Image (Apple Pie) */
+#hero-image img {
+  width: 100%;
+  height: 715px;
+}
+
+.hero-text {
+  color: #fff;
+  font-size: 5em;
+  position: absolute;
```

Comment:
Position absolute using pixels will make your web-page unresponsive.  Not the focus of this assignment but keep in mind for future reference.